rsnapshot-program.pl

```diff
@@ -6716,6 +6739,28 @@ =head1 CONFIGURATION
 
 =back
 
+B<include_conf_dir>   Include any files in this directory at this point, defaults ending in .conf
```

Comment:
> Include any files in this directory at this point, defaults ending in .conf

Not very clear. It's not including "any files". Suggest replace with:

Include any files in this directory at this point. By default only files ending in .conf are included.